src/components/UserPlan.tsx

```diff
@@ -1,42 +1,206 @@
+    planType,
     itemDescription,
     buttonUrl,
     buttonText,
     pendingFreeUpgrade,
     remainingCredits,
     description,
+    isSubscribers,
+    isMonthlyByEmail,
   } = plan;
 
+  const {
+    smsEnabled,
+    remainingCredits: smsRemainingCredits,
+    description: smsDescription,
+    buttonUrl: smsButtonUrl,
+    buttonText: smsButtonText,
+  } = sms;
+
+  const intl = useIntl();
+  const planPrepaid = intl.formatMessage({ id: "header.plan_prepaid" });
+  const sendRequest = intl.formatMessage({ id: "header.send_request" });
+  const profile = intl.formatMessage({ id: "header.profile" });
+  const sendMails = intl.formatMessage({ id: "header.send_mails" });
+  const enabled = intl.formatMessage({ id: "header.enabled" });
+  const suscribers = intl.formatMessage({ id: "header.plan_suscribers" });
+  const emails = intl.formatMessage({ id: "header.plan_emails" });
+  const availables = intl.formatMessage({ id: "header.availables" });
+  const toolTipLastPlanText = intl.formatMessage({
+    id: "header.tooltip_last_plan",
+  });
+
+  const isPlanTypeMonthlyDeliveries = planType === "monthly-deliveries";
+  const isPlanTypeSuscribers = planType === "suscribers";
+
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const openModalHandler = () => {
+    setIsModalOpen(true);
+  };
+
+  const showTooltip = () => (
+    <Tooltip>
+      <OpenModalButton
+        className="user-plan close-user--menu dp-tooltip-left"
+        openModalHandler={openModalHandler}
+      >
+        {sendRequest}
+        <div className="tooltiptext">{toolTipLastPlanText}</div>
+      </OpenModalButton>
+      <span className="ms-icon icon-info-icon"></span>
+    </Tooltip>
+  );
+
+  const renderPlanLink = () => {
+    if (buttonUrl && !pendingFreeUpgrade)
+      return (
+        <a className="user-plan" href={buttonUrl}>
+          {buttonText}
+        </a>
+      );
+  };
+
   return (
     <div className="user-plan--container">
-      <div className="user-plan--type">
-        <p className="user-plan--monthly-text">
-          <strong>{planName}</strong> ({maxSubscribers} {itemDescription})
-        </p>
-        {buttonUrl && !pendingFreeUpgrade && (
-          <a className="user-plan" href={buttonUrl}>
-            {buttonText}
-          </a>
-        )}
-        <p className="user-plan--monthly-text" />
-      </div>
-
-      <div className="user-plan--type">
-        <div className="user-plan--buyContainer">
-          <p>
-            <strong>{remainingCredits}</strong> {description}
-          </p>
-        </div>
-      </div>
+      {!hasClientManager && (
+        <UserPlanType>
+          {isSubscribers || isMonthlyByEmail ? (
+            <>
+              <MonthlyPlan>
+                <strong>{planName}</strong> ({maxSubscribers} {itemDescription})
+              </MonthlyPlan>
+              {renderPlanLink()}
+            </>
+          ) : (
+            <>
+              <MonthlyPlan>{planPrepaid}</MonthlyPlan>
```

Comment:
Relacionado [a este comentario](https://github.com/FromDoppler/doppler-menu-mfe/pull/87/files#r895654854), creo que en general, esta es la forma usada:

```suggestion
              <MonthlyPlan><FormattedMessage id="header.plan_prepaid" /></MonthlyPlan>
```

Reply:
https://formatjs.io/docs/react-intl/components#formattedmessage